samples/cpp/speech_sample/main.cpp

```diff
@@ -243,7 +243,7 @@ int main(int argc, char* argv[]) {
         };
         gnaPluginConfig[ov::intel_gna::execution_target.name()] = parse_target(FLAGS_exec_target);
         gnaPluginConfig[ov::intel_gna::compile_target.name()] = parse_target(FLAGS_compile_target);
-        gnaPluginConfig[ov::intel_gna::memory_reuse.name()] = false;
+        gnaPluginConfig[ov::intel_gna::memory_reuse.name()] = !FLAGS_disable_compact_mode;
```

Comment:
I would suggest using the new name of this option - memory_reuse, with possible values (on/off or true/false)

Reply:
renamed the option